factcast-spring-boot-autoconfigure/src/main/java/org/factcast/spring/boot/autoconfigure/snap/RedissonSnapshotCacheAutoConfiguration.java

```diff
@@ -32,8 +31,7 @@
 @ConditionalOnClass({RedissonSnapshotCache.class, RedissonClient.class})
 @ConditionalOnMissingBean(SnapshotCache.class)
 @Import({RedissonSnapshotProperties.class})
-@Generated
-@AutoConfigureOrder(-100)
+@AutoConfigureBefore(FactCastSnapshotCacheAutoConfiguration.class)
 public class RedissonSnapshotCacheAutoConfiguration {
 
```

Comment:
The comment "// compacting no longer needed with redis as we use EXPIRE instead" can go away then right? Sadly I cannot put an suggestion on it. Github is broken :D